test/test_preprocess.py

```diff
@@ -8,10 +8,10 @@
 file_path = os.path.dirname(os.path.abspath(__file__))
 config_path = os.path.sep.join(file_path.split(os.path.sep)[:-1])
 sys.path.append(config_path)
-from connlp.preprocess import EnglishTokenizer, Normalizer
+from connlp.preprocess import EnglishTokenizer, Normalizer, KoreanTokenizer
 tokenizer = EnglishTokenizer()

```

Comment:
Feel free to modify the existing codes written by the other contributors.
Suggest renaming the "tokenizer" as "eng_tokenizer."